build-profiling-ffi.sh

```diff
@@ -19,16 +19,17 @@ ARG_FEATURES=""
 run_tests=true
 
 usage() {
-    echo "Usage: `basename "$0"` [-h] [-f FEATURES] [-T] dest-dir"
+    echo "Usage: `basename "$0"` [-h] [-f FEATURES] [-t TRIPLET] [-T] dest-dir"
     echo
     echo "Options:"
     echo "  -h          This help text"
     echo "  -f FEATURES Enable specified features (comma separated if more than one)"
+    echo "  -t TRIPLET  Target triplet to build for, defaults to host triplet"
     echo "  -T          Skip checks after building"
     exit $1
 }
 
-while getopts f:hT flag
+while getopts f:ht:T flag
 do
     case "${flag}" in
         f)
```

Comment:
Two notes here:

* I think we're misusing the shift -- it actually breaks if we try to pass more than one setting now; e.g. `-f foo -t triplet` or `-T -t triplet`. My bash-foo isn't great but [this suggests](https://unix.stackexchange.com/questions/484167/shift-in-getopts-loop-clarification-needed) how we may fix it. I think it has "worked" until now since nobody tried to use `-f` with  `-T`, but it seems reasonable to want to combine `-f` and `-t` for instance?

* As a script UX thing, consider using something other than `-t` for the triplet, to avoid the collision with the test. Maybe `-a` (arch?) or `-c` (cpu) or something like that? Alternatively, rename the `-T` to something else, I think that works too :)

Reply:
Updated the script
- `./build-profiling-ffi.sh -s -t aarch64-apple-darwin ../libdatadog-ffi`
- `./build-profiling-ffi.sh -t x86_64-apple-darwin ../libdatadog-x86-64`
- `./build-profiling-ffi.sh -t x86_64-apple-darwin -s ../libdatadog-x86-64`

These work, so multiple flags work and updated the skip tests to s